src/rest/main/main.go

```diff
@@ -55,7 +54,7 @@ func init() {
 	flag.StringVar(&certFile, "cert", "", "Server certificate file path")
 	flag.StringVar(&keyFile, "key", "", "Server private key file path")
 	flag.StringVar(&caFile, "cacert", "", "CA certificate for client certificate validation")
-	flag.StringVar(&clientAuth, "client_auth", "none", "Client auth mode - none|cert|user|jwt")
+	flag.Var(server.ClientAuth, "client_auth", "Client auth mode(s) - cert,user,jwt")
```

Comment:
Could we change "user" authentication to "password" authentication instead? It's just a nomenclature thing, but I think it's more accurate to describe username/password authentication as password-based authentication, since even in the other cases (cert, JWT), you're technically authenticating a _user_ as well.

Reply:
Sorry, I know this means you'll have to change it everywhere else too.

Reply:
change, see latest commit.